### ISSUE TITLE
"Failed to publish: Request queued (action evaluate) but server did not respond" error when trying to publish the review comments

### ISSUE THREAD

**Issue by daria-sorokina-da** [Reviewable/Reviewable#1182]
After adding some comments and clicking "Publish and Block" getting an error message "Failed to publish: Request queued (action evaluate) but server did not respond", and the comments do not get saved.

**Comment by daria-sorokina-da** [comment]
Relogin helped fix the issue

**Comment by pkaminski** [comment]
Hey Daria, glad to hear to you got things sorted out.  Typically this error message indicates an overloaded server but that definitely wasn't the case in this instance.  I'm surprised that logging out and back in made a difference, though — is that all you did, or did you close / reopen the page as well?

**Comment by daria-sorokina-da** [comment]
@pkaminski Hello!
Closing/reopening of the page did not help, so I logged out and logged in again.
I'm not 100% sure that it was what helped as just 5 mins ago reviewable worked ok without having to log in. But then for 5 mins straight all of my attempts to save comments resulted in this error. 
It could be that the server had an outage at that moment, and then it stopped right at the moment when I re-logged in :)
